### DISCUSSION TITLE
antd是否有打点上报日志逻辑

### DISCUSSION THREAD

**Discussion by qq1037765897** [ant-design/ant-design#43587] [Q&A]
这个包被判定危险的npm包了

**Comment by li-jia-nan** [comment]
哪里判定危险了？我这边去 npm 看了一下好像没啥问题

**Reply by qq1037765897** [reply]
说是有打点日志，会泄露域名，我看了下源码是没发现有打点逻辑

**Reply by li-jia-nan** [reply]
组件里面是没有日志的，官网文档里有埋点

**Comment by qq1037765897** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by zombieJ** [comment]
1. 没有该功能，需要埋点请自行对接相关服务。
2. 如果判定有风险，可以问问相关检测服务的判断依据是什么，应该是误报。

**Reply by qq1037765897** [reply]
好的 应该是误报，感谢~

**Comment by afc163** [comment]
> antd是否有打点上报日志逻辑

没有。

> 这个包被判定危险的npm包了

如何判定的呢？依据是什么

**Reply by qq1037765897** [reply]
应该是误报，感谢~
